doc/userguide/rules/tls-keywords.rst

```diff
@@ -116,6 +116,21 @@ Usage::
 
   tls_cert_valid;
 
+tls_cert
+--------
+
+Do a "raw" match on the TLS certificate. This keyword only matches on the
+first certificate in the certificate chain.
+
+Example::
+
+  alert tls any any -> any any (msg:"match bytes in TLS cert"; tls_cert; \
+    content:"|06 09 2a 86|"; sid:200070;)
+
+``tls_cert`` is a 'Sticky buffer'.
```

Comment:
**s**ticky instead of **S**ticky? :) but in general doc looks good

Reply:
It is written as 'Sticky' on all the other keywords in the same file. Do you want me to lowercase the existing keywords as well? :)

Reply:
I like the lowercase one better as well. If you want to update it elsewhere, please do.